testing/ubuntu/presubmit/run_if_changed.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+### Ignore this test if there are no relevant changes
+cd ${CMLE_REPO_DIR}/${CMLE_TEST_BASE_DIR}
+
+DIFF=`git diff master $KOKORO_GITHUB_PULL_REQUEST_COMMIT $PWD`
```

Comment:
@elibixby Kokoro will still run tests for unchanged sample tests? So if I only change flowers tests we will run tests for all samples is how I read this code. Correct?